core/src/main/java/org/jdbi/v3/core/mapper/InternetMapperFactory.java

```diff
@@ -37,10 +37,10 @@
  *     <li>{@link URI}</li>
  * </ul>
  */
-class InternetMapperFactory implements ColumnMapperFactory {
+public class InternetMapperFactory implements ColumnMapperFactory {
```

Comment:
neither of these public should be needed. The actual mapper can be retrieved with `ctx.getConfig(ColumnMappers.class).forType(InetAddress.class)` if needed and the actual type is an implementation detail that can easily be hidden. No user should ever have the need to actually instantiate the mapper directly outside a test (and those can be package private).